### DISCUSSION TITLE
Apply non-random affine

### DISCUSSION THREAD

**Discussion by Constantin-Jehn** [TorchIO-project/torchio#870] [Q&A] [answered]
Hello,

what would you suggest as simplest way to apply a specific affine to an image.
That would be the deterministic pendent of RandomAffine.

Thanks a lot.

**Comment by fepegar** [comment, marked as answer]
Hi, @Constantin-Jehn. You can use `Affine` for that:

```python
import torchio as tio
colin = tio.datasets.Colin27().t1
colin.plot()
```

![Figure_1](https://user-images.githubusercontent.com/12688084/168486046-c5d2f7c8-6e2d-4c5d-96b5-3feafe560f1e.png)

```python
look_up = tio.Affine(1, (45, 0, 0), 0)
colin_looking_up = look_up(colin)
colin_looking_up.plot()
```

![Figure_12](https://user-images.githubusercontent.com/12688084/168486061-79a2432b-661c-4152-90c7-20c2a35fbb87.png)

**Reply by Constantin-Jehn** [reply]
Hi,
thanks a lot for your answer!

I would suggest to add the transform to torchio's documentation, at least I would have expected to find it either under Transforms/PreProcessing/Spatial or Transforms/Others.

Is there also a was to provide the affine-matrix itself as input?

Cheers

**Reply by fepegar** [reply]
Hi, @Constantin-Jehn. Sorry for the delayed reply. There is currently no way to pass an affine matrix directly.

I think it would be nice to mention in the docs that there are non-random versions of all transforms. Where do you think would be a good place for that?
